packages/website/docs/autocomplete-js.md

```diff
@@ -142,24 +142,24 @@ autocomplete({
 });
 ```
 
-### `renderEmpty`
+### `renderNoResults`
 
-> `(params: { root: HTMLElement, state: AutocompleteState<TItem>, sections: VNode[], createElement: Pragma, Fragment: PragmaFrag }) => void`
+> `(params: { children: VNode, state: AutocompleteState<TItem>, sections: VNode[], createElement: Pragma, Fragment: PragmaFrag }) => void`
 
-Function called to render an empty section when no hits are returned. It is useful for letting the user know that the query returned no results.
+Function called to render a no results section when no hits are returned. It is useful for letting the user know that the query returned no results.
```

Comment:
```suggestion
Function called to render a `no results` section when no hits are returned. It is useful for letting the user know that the query returned no results.
```

I feel like this should be "escaped" as it's more a naming than a term, not sure if you see what I mean

Reply:
Yes I agree—the docs team is gonna rewrite all the API refs so I'm waiting for them to judge what's best 🙂